pkg/serverhandler/handlertest/test_service_project.go

```diff
@@ -90,6 +90,24 @@ func TestServiceProject(t *testing.T, factory Factory) {
 			require.Len(resp.Projects, 1)
 		}
 	})
+
+	t.Run("destroy", func(t *testing.T) {
+		require := require.New(t)
+
+		// Destroys the specified project
+		{
+			_, err := client.DestroyProject(ctx, &pb.DestroyProjectRequest{
+				Project: &pb.Ref_Project{Project: "example"},
+			})
+			require.NoError(err)
+
+			resp, err := client.GetProject(ctx, &pb.GetProjectRequest{
+				Project: &pb.Ref_Project{Project: "not-found"},
```

Comment:
We should be getting the project with ID "example" to test that project "example" was successfully destroyed.

We should also show two different tests, one that shows resources are not deleted with the destroy resources flag, and one that shows they are successfully deleted without the flag.

Reply:
@xiaolin-ninja added! I had to implement it in the `internal/runner` package as the option to skip destroying resources is available when creating the project destroy job only, not here in `service_project`.